omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml

```diff
@@ -11,203 +11,352 @@
           formDisplayStyle="Standard"
           formLabel="Maternity">
 
-    <style>
-        form label, .form label {
-            float: left;
-            padding: 0px 10px 0 0;
-        }
-
-        form input[type="checkbox"], form input[type="radio"], .form input[type="checkbox"], .form input[type="radio"] {
-            margin: 10px 15px;
-        }
-
-        table td {
-            background: #FFFFFF;
-            color: #404040;
-        }
-
-        form select, .form select {
-            min-width: 0px;
-        }
-
-        form input, form select, form textarea, form ul.select, .form input, .form select, .form textarea, .form ul.select {
-            background-color: #FBFBFB;
-            border: 1px solid #CEC9C9;
-            border-radius: 5px;
-            margin-bottom: 10px;
-            padding: 10px;
-            min-width: 50%;
-        }
-
-        .row {
-            width: 100%;
-            float: left;
-            display: inline-flex;
-            clear: both;
-        }
-
-		.row .row-col2 {
-			float: left;
-			width: 50%;
-		}
-
-        table th, table td {
-            border: 1px solid #C7C7C7;
-            min-width: 10%;
-        }
-
-        table {
-            margin: 20px 0px;
-        }
-
-		fieldset{
-			width:97%;
-			margin:0 0 30px 0
-		}
-		
-		fieldset legend{
-			font-weight:600;
-		}
-
-		fieldset table {
-			width:100%;
-			margin: 0px;
-		}
-
-        fieldset table td {
-            width: 25%;
-            vertical-align: top;
-        }
-
-		fieldset table td > p {
-			margin: 0px;
-		}
-
-		fieldset table td input:text {
-			width: 100%;
-		}
-
-		fieldset table td label {
-			padding: 0px 10px 0 0;
-		}
-				
-		select.hfe-hours, select.hfe-minutes, select.hfe-seconds {
-			 width: 28px;
-			 min-width: 65px;
-			 float: left;
-			 position: relative;
-			 top: -58px;
-			 left: 244px;
-		}
-		
-		
-    </style>
+	<style>

```

Comment:
@moshonk Can these styles be moved to the aijar.css so that they are not duplicated across forms